tutorials/vpc-public-gateway-autoconfiguration/index.mdx

```diff
@@ -0,0 +1,188 @@
+excerpt: |
+  VPC consists of Private Networks and Public Gateways to create an isolated network environment for your instances, without exposing them on the public Internet. 
+totalTime: PT20M
+  - network
+brands:
+  - vpc
+keywords: cloud tutorials, cloud documentations, cloud instance, cloud hosting, network, virtual instance, private network, vpc, public gateway, virtual private cloud
+---
+
+## VPC - Overview
+
+VPC (Virtual Private Cloud) provides network functionalities for your Scaleway cloud. VPC products include [Private Networks](https://www.scaleway.com/en/private-networks/) , enabling you to build a virtual L2 network between your instances, and the [Public Gateway](https://www.scaleway.com/en/public-gateway/), enabling IP autoconfiguration of your Private Networks and their communication with the Internet.
+
+<Message type="requirement">
+
+- You have an account and are logged into the [Scaleway Console](https://console.scaleway.com)
+- You have [configured your SSH Key](/console/my-project/how-to/create-ssh-key/)
+- You have created a [Virtual Instance](https://www.scaleway.com/en/virtual-instances/)
+
+</Message>
+
+## Creating a Private Network for the Virtual Instance
+
+1. Connect to your Instance using [SSH](https://www.scaleway.com/en/docs/compute/instances/how-to/connect-to-instance/). 
+
+  ```
+  ssh root@<your_instance_ip>
+  ```
+
+  <Message type="tip">
+
+  You can connect to your Instance even if it does not have a [flexible IP address](https://www.scaleway.com/en/docs/compute/instances/how-to/use-flexips/) by using its IPv6 address.
+
+  </Message>
+
+2. Check the status of your network interfaces using the `ip a` command. You see the details of the default Ethernet adapter of your Instance:
+
+  ```
+  2: ens2: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc fq_codel state UP group default qlen 1000
+    link/ether de:1c:a4:2b:d0:39 brd ff:ff:ff:ff:ff:ff
+    inet 10.70.104.113/31 scope global dynamic ens2
+       valid_lft 85983sec preferred_lft 85983sec
+    inet6 2001:bc8:634:c38::1/64 scope global
+       valid_lft forever preferred_lft forever
+    inet6 fe80::dc1c:a4ff:fe2b:d039/64 scope link
+       valid_lft forever preferred_lft forever
+  ```
+
+3. Go to the [VPC section](https://console.scaleway.com/vpc/private-networks) of the Scaleway console. The VPC creation page displays. 
+
+  <Lightbox src="scaleway_vpc_splash.png" alt="" />
+
+4. Click **Create a new Private Network**. The Private Network creation wizard displays. 
+
+5. Enter the details of the new Private Network and click **Create a Private Network**:
+
+    - Enter a **Name** for the Private Network
+    - Select the region for the Private Network
+
+    <Message type="note">
+
+    Make sure to create the Private Network in the same region as your Instance.
+
+    </Message>
+
+6. The list of your Private Networks displays. Click on the Private Network's name to display its information. 
```

Comment:
I think "The list of your private networks displays" should not be a step. Peut-être mettre la phrase avant la step  6 ?